aeg.html

```diff
@@ -0,0 +1,34 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet"  href="style.css">
+</head>
+<body>
+    
+    <h1>Alpha Existential Graphs</h1>
+    <div class="container">
+        <button class="btn btn1">Cut</button>
+        <button class="btn btn2">Draw</button>
+        <button class="btn btn3">Paste</button>
+        <button class="btn btn4">Copy</button>
+    </div>
+    
+    <div include-html="components/navbar.html"></div>
+<ul class="content home">
+<nav>
+    <div> 
+        <li><a href="/index.html" >Homepage</a></li>     
+        <li><a href="/aeg.html">Alpha Existential Graphs</a></li>
+        
+        
+        
+        
```

Comment:
Pointless Whitespace? Presumably this is for more links but you should add those lines when you get to it, not leave this empty.